packages/smst-to-html/rollup.config.js

```diff
@@ -0,0 +1,7 @@
+import getConfig from "../../rollupUtils";
+
+const config = process.env.FORMAT === 'umd'
+	? getConfig('toHtml.ts', 'umd', 'umd', ['@suitest/smst'], {'@suitest/smst': '@suitest/smst'})
```

Comment:
hard to read functions with many arguments, please consider change it for accepting single settings object.